packages/nextjs/hooks/scaffold-stark/__tests__/mocks/useScaffoldStarkProfile.mock.ts

```diff
@@ -0,0 +1,16 @@
+import { StarkProfile } from "starknet";
+import { vi } from 'vitest';
+
+const mockProfile: StarkProfile = {
+  name: 'Test User',
+  profilePicture: 'https://example.com/profile.jpg',
+  discord: 'testuser#1234',
+  twitter: 'testuser_twitter',
+  github: 'testuser_github',
+  proofOfPersonhood: true,
+};
+
+export const mockUseScaffoldStarkProfile = vi.fn().mockReturnValue({
```

Comment:
yes, because u are mocking the very thing you tried to test haha, that is why. 

mocks are supposed to be only for components within the function (e.g. some util function that is used) that you are sure of the input and output values.

Reply:
@metalboyrick haha my bad, my head was on christmas already haha should be good now!